frontend/public/components/mounted-vol.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import * as _ from 'lodash-es';
+
+import {
+  getVolumeType,
+  getVolumeLocation,
+  getVolumeMountPermissions,
+  getVolumeMountsByPermissions,
+} from '../module/k8s/pods';
+import {
+  VolumeIcon,
+  ResourceIcon,
+} from './utils';
+
+const ContainerLink = ({name, pod}) => <span className="co-resource-link co-resource-link--inline">
+  <ResourceIcon kind="Container" />
+  <Link to={`/k8s/ns/${pod.metadata.namespace}/pods/${pod.metadata.name}/containers/${name}`}>{name}</Link>
+</span>;
+
+const Volume = ({volumeMounts, pod}) => {
+  const kind = _.get(getVolumeType(volumeMounts.volume), 'id', '');
+  const loc = getVolumeLocation(volumeMounts.volume);
+  const name = volumeMounts.name;
+  const mountPermissions = getVolumeMountPermissions(volumeMounts);
+
+  return <div>
+    <div className="row">
+      <div className="col-sm-2 col-xs-3">{name}</div>
+      {volumeMounts.mounts.map((m, i) => <React.Fragment key={i}>
+        <div className="col-sm-2 col-xs-3">{_.get(m, 'mountPath', '-')}</div>
+        <div className="col-sm-2 col-xs-3">{_.get(m, 'subPath', '-')}</div>
+      </React.Fragment>)}
```

Comment:
This seems wrong. If there are multiple mounts, the columns will no longer match the headers, correct?

Reply:
@spadgett correct I agree the col will not match the headers if we have multiple mount path or the longer string name.

So can I hide mount path and subpath when it increases the column size and show only name, type, permission and utilized by?

Reply:
Yeah I think it makes sense to only have those columns at wider screen widths.

Reply:
Ack will do the required changes.

Reply:
It looks like this is still an issue?

Reply:
@ 150% screen size
![screenshot from 2019-01-11 20-46-31](https://user-images.githubusercontent.com/6695156/51042376-217ad100-15e2-11e9-8027-91b6126c1df2.png)
@ 100% screen size
![screenshot from 2019-01-11 20-46-08](https://user-images.githubusercontent.com/6695156/51042377-22136780-15e2-11e9-8e7a-93de8d9aa46b.png)

Reply:
@spadgett sorry I didn't understand. Do you want to add Overflow component for the Type column also?

Reply:
It's showing the wrong data in the wrong column. You're creating new columns in a loop in the middle of rending a row, so the column headers don't match the content.

Reply:
@spadgett Ahh, got the problem. Overflow component has it's `-` if the value not there and I was also adding `-` if the value is not there. So the second hyphen was getting shifted to another row.

Reply:
I fixed it from my side. can you test it once or can you provide me setup where I can test? I was not able to replicate the issue because I didn't have the monitoring deployed.

I tried deploying using the [prometheus-operator](https://github.com/coreos/prometheus-operator/tree/master/contrib/kube-prometheus) but the prometheus-k8s stateful set was not created in monitoring namespace.

Reply:
![screenshot from 2019-01-15 16-02-37](https://user-images.githubusercontent.com/6695156/51175027-35268000-18df-11e9-95ce-8cda23b03b42.png)

@spadgett Fixed the issue. Please have a look.